lib/screens/wallet/wallet_actions.dart

```diff
@@ -56,14 +56,20 @@ class _WalletActionsState extends State<WalletActions> {
     final isWalletReady = context.select((WalletState state) => state.ready);
     final showActionButton = !walletActionsLoading && isWalletReady;
     final actionButton = context.select(selectActionButtonToShow);
-    int pluginsCount = wallet!.plugins.length;
-    PluginConfig? onePlugin = pluginsCount < 1 ? null : wallet.plugins[0];
+    int pluginsCount = wallet?.plugins.length ?? 0;
+    PluginConfig? onePlugin = pluginsCount < 1 ? null : wallet!.plugins[0];
```

Comment:
If it's `wallet?.plugins.length` on the previous line but `wallet!.plugins[0]` on this line, seems unsafe.

Maybe:
`PluginConfig? onePlugin = pluginsCount < 1 ? null : wallet?.plugins[0] ?? null;` ?